_content/news/_posts/2018-10-19-kerstkampen.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title:  "Basketbal voor de jeugd tijdens de kerstperiode"
+cover: /BasketLummen/Website/_content/albums/2018-10-13-Basket-Lummen-DSEA-Sint-Katelijne-Waver-DSEA/_thumbnails/IMG_6412.jpg
+date:   2018-10-25 20:00:00
+description: Basketbal voor de jeugd, ook tijdens de kerstperiode!.
+permalink: /news/vervang-dit-door-de-bestandsnaam-zonder-extensie/
+---
+
+Ook tijdens de kerstperiode zullen de jeugdspelers van Basket Lummen leerrijke ervaringen kunnen opdoen. Er is niet alleen het FROS-kamp, met coaches van Basket Lummen, maar ook het Bounce Camp Hans Vanwijn, een voormalig speler van Basket Lummen. Verder volgt er nog een evenement (Brecht)..... details...?
+
+## FROS-kamp
+
+Locale coaches, zoals Jolien Wouters, Janne Wouters en Caro Swolfs, hebben een ijzersterk concept voor de beginnende en gevorderde basketter. Het kamp is al jarenlang een vaste waarde in Limburg en bundelt heel wat ervaring. Het wordt georganiseerd door Bart Wouters, die onze DSE-A-ploeg ondersteunt.
+
```

Comment:
Hier zou ik de volgorde omdraaien:

Het FROS kamp is al jarenlang een vaste waarde in Limburg en bundelt heel wat ervaring. FROS heeft een ijzersterk concept neergezet voor de beginnende en gevorderde basketter, het wordt mede-georganiseerd door Bart Wouters, die ook onze Dames A ondersteunt. In dit concept werken ze met lokale coaches, waaronder ook speelsters van de Dames A: Jolien Wouters en Caro Swolfs,